isis/src/rosetta/apps/rosvirtis2isis/RosettaVirtisMHousekeeping.def

```diff
@@ -0,0 +1,107 @@
+dataSCET, double, {0.0, 1.0, 0.0}
```

Comment:
@scsides Do you think it's reasonable to make this a .def file?

Reply:
works for me, it's original name didn't have solid meaning